templates/kubernetes/docs/metallb.md

```diff
@@ -74,32 +78,14 @@ your environment. The IP addresses can be specified as a range, such as
 "192.168.1.88-192.168.1.89", or as a comma-separated list of pools in CIDR
 notation, such as "192.168.1.240/28, 10.0.0.0/28".
 
-Configuring the IP addresses can be done either at time of deployment via a
-[bundle overlay][], or later by changing the charm config via Juju.
-
-An example bundle overlay might look like:
-
-```yaml
-applications:
-  metallb-controller:
-    options:
-      iprange: "192.168.1.88-192.168.1.89"
-```
-
-You would then specify this when deploying the bundle:
-
-```bash
-juju deploy metallb --overlay ./overlay.yaml
-```
-
-Alternatively, you can change the config directly on the metallb-controller
-charm at any time:
+Configuring the IP addresses can be done at any time by changing the charm config via Juju.
 
 ```bash
 juju config metallb-controller iprange="192.168.1.240/28, 10.0.0.0/28"
 ```
 
 ### Note: Using RBAC
+<a name="rbac-note"> </a>
```

Comment:
Same here

Reply:
this is the only one that was actually referenced as far as I can tell. thanks. There are loads more scattered amongst the pages but i will make a card to tidy them up